doc/content/help/faq.md

```diff
@@ -1,14 +1,136 @@
 # Frequently Asked Questions
 
-#### What if the `git clone` command does nothing?
+
+#### 1. What if the `git clone` command does nothing?
 
 Depending on your network configuration the HTTP access to the repository may not work. In this case
 you must use [SSH](https://en.wikipedia.org/wiki/Secure_Shell) keys. First, follow the instructions
 to
 [generate a new SSH key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#generating-a-new-ssh-key)
 and then run the following clone command.
 
-
 ```
 git clone git@github.com:idaholab/mastodon.git
 ```
+
+#### 2. How to `update_and_rebuild_libmesh.sh` in parallel?
+
+In the MASTODON folder, run the following command.
+
+```
+JOBS=8 ./moose/scripts/update_and_rebuild_libmesh.sh
+```
+
+The number `8` above is the number of processors and can be changed to a different value.
+
+#### 3. How to run MASTODON on an INL high performance computing account?
+
+There are several steps in this process:
+
+- Log in to your hpc account using `ssh USERNAME@HPC_NAME`. An example is,
+
+```
+ssh john@falcon1
+```
+
+- Repeat the MASTODON installation process explained in [Getting Started/MacOS (Mojave)](https://mooseframework.inl.gov/mastodon/getting_started/macos_mojave.html) or similar. That is, modify your bash profile, install MASTODON, update and rebuild libmesh, and compile and test MASTODON. Note that you have to setup you [github](https://github.com) on the hpc.
+
+- Now you can run an example using a special file with a `.sh` extension. For example, if you want to run [Example 03](https://mooseframework.inl.gov/mastodon/examples/example3.html) on hpc, create a file named `Example03_hpc.sh`. The filename can be anything but it should end with a `.sh` extension. This file should contain the following lines.
```

Comment:
Job scheduling is performed using the Portable Batch System (PBS). Batch script files should be created with a unique name using the .sh extension. An example is shown below: 

Batch files can be submitted into the queue  using the qsub command as shown below:

For more information on PBS see http://hpcweb.inl.gov/home/pbs  (I am not sure if we can put this link here or not...)